css: fixed search message alert

amandaalexandre opened the pull request:
- Fixed the border to let it grow when the text is wrapped
- Used inline-flex property to align elements inside.

As a default, the jQuery function .show() gives the div display: block.

But we need it to be inline-flex if we're not to create another div just to align the elements inside.
There are several ways of doing this, so if it's not apropriate, let me know. I tried to avoid using .css() in the JS file to alter the styling. 

If the solution adopted (changing the display to inline-flex in the CSS file) is not ideal, let me know and I can just add a buffer div between the parent div and its elements just to align them as usual.

I apologized for the duplicate PR but the previous one had a long commit history that I couldn't squash due to merge conflicts with a file I didn't know. For safety, I decided to revert to a previous commit and push a clean history. (Now I learned I can create PR drafts =) ) 

Fixes: [<!-- Issue link, or clear description.-->](https://github.com/zulip/zulip/issues/23273)

**Screenshots and screen captures:**
Before:
![image](https://user-images.githubusercontent.com/15349795/197750852-a5a98895-9aaa-4d30-9e4e-d29e5a13cffc.png)

After:
![Opera Instantâneo_2022-10-25_072145_demo-elih1848 zulipdev com](https://user-images.githubusercontent.com/15349795/197750996-8634a7ee-7c19-40af-a5f3-ef5b94ca544d.png)

**Self-review checklist**

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [ ] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [x] Explains differences from previous plans (e.g., issue description).
- [x] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)).

- [ ] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [ ] Strings and tooltips.
- [ ] End-to-end functionality of buttons, interactions and flows.
- [ ] Corner cases, error conditions, and easily imagined bugs.

alya commented:
Looks good to me in manual testing! @amanagr would you be up for reviewing this one?

amanagr commented:
@amandaalexandre thanks for working on this. I removed all your changes and just deleted the CSS which applied the `height` restriction to the notices. This seems to working well be to me.

<img width="361" alt="Screenshot 2022-10-27 at 3 01 37 PM" src="https://user-images.githubusercontent.com/25124304/198249375-591f2306-0acc-4911-bf8a-b2bc5e7e3eff.png">

Please read [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages) to write commit messages in the correct format for your future PRs.

amandaalexandre commented:
> @amandaalexandre thanks for working on this. I removed all your changes and just deleted the CSS which applied the `height` restriction to the notices. This seems to be working well.
> 
> <img alt="Screenshot 2022-10-27 at 3 01 37 PM" width="361" src="https://user-images.githubusercontent.com/25124304/198249375-591f2306-0acc-4911-bf8a-b2bc5e7e3eff.png">
> 
> Please read [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages) to write commit messages in the correct format for your future PRs.

Thank you for your feedback, @amanagr ! I added some alignment on the text per request in the former PR. I also think it looks good the way it is.

amanagr commented:
@alya LGTM

alya commented:
Hm, the PR description screenshot aligns the second line of text with the first line (to the right of the `!`), which looks better to me than the latest screenshots. Can we pleas go back to that design? Thanks!

amanagr commented:
<img width="361" alt="Screenshot 2022-10-29 at 9 13 08 AM" src="https://user-images.githubusercontent.com/25124304/198812405-a9a23dc6-cbe8-457c-8a22-d8e5d0eb28ef.png">

@alya  updated!

alya commented:
Looks great to me in manual testing!

@timabbott This PR has been reviewed / amended by @amanagr ; tagging for your review.

timabbott commented:
Merged, thanks for the contribution @amandaalexandre!
